appveyor.yml

```diff
@@ -22,3 +22,12 @@ artifacts:
   - path: Watchman
     name: watchman
     type: zip
+
+environment:
+  matrix:
+    - WATCHMAN_WIN7_COMPAT: 1
+    - WATCHMAN_WIN7_COMPAT:
+
+platform:
+  - x86
```

Comment:
I had only planned on supporting 64-bit native systems, and I think much of our win32 build setup is just using 64-bit stuff anyway.   I'm not opposed to taking in patches to keep things working on 32-bit systems, but we won't be putting in any real effort to keeping that working.  If you don't need 32-bit support, it would be nicer on the appveyor infra to remove this platform from the matrix :-)